phpBB/phpbb/install/schemas/schema_data.sql

```diff
@@ -0,0 +1,821 @@
+#
```

Comment:
I wonder if these files shouldn't sit in /phpBB/install/schemas (so /phpBB/install_new/schemas for now)

@phpbb/development-team thought?